<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configw</class>
 <widget class="QDialog" name="configw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resouse.qrc">
    <normaloff>:/image/bank.ico</normaloff>:/image/bank.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
font: 25 10pt &quot;微软雅黑 Light&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(107, 179, 200, 255), stop:1 rgba(255, 255, 255, 255));
};

</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>服务策略</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="maxseqlen_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="maxrest_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="minsec_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="maxsec_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="minrest_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="vipservlen_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="maxseqlen">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最大队列长度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>MaxSeqLen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="minsec">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最短服务时长&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>MinSec</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="maxsec">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最长服务时长&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>MaxSec</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="maxrest">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最长休息时长&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>MaxRestSec</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="vipservlen">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;动态关闭vip窗口检查时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>VIPSERVLen</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="minrest">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最短休息时长&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>MinRestSec</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>（s）</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>（s）</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>（s）</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>（s）</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <property name="text">
       <string>（s）</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;
</string>
      </property>
      <item>
       <property name="text">
        <string>VIP窗口顺便服务策略（默认）</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VIP窗口定向服务策略</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(107, 179, 200, 255), stop:1 rgba(255, 255, 255, 255));

</string>
   </property>
   <property name="text">
    <string>确认（&amp;Y）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(107, 179, 200, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
   <property name="text">
    <string>取消（&amp;N）</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resouse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>configw</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
